build.gradle

```diff
@@ -243,6 +245,16 @@ task copyToRoot(type: Copy, dependsOn: [copyToLibs, jar, converterJar]) {
     rename "(.+)-$version(.+)", '$1$2'
 }
 
+task unixLineEndings( type: Exec) {
+    if(Os.isFamily(Os.FAMILY_WINDOWS)){
+        workingDir "$projectDir/code/sbin"
+        executable "bash"
+        args "-c", "fromdos $projectDir/code/pcgen.sh"
+    }else if(Os.isFamily(Os.FAMILY_UNIX)){
+        commandLine "dos2unix", "$projectDir/code/pcgen.sh"
+    }
+}
+
```

Comment:
It does give output odf the need for this tool, also it has been requested to be installed in our server.

I'm all for it if it can be done via setting on Git, this was the only way I could make it work.